Corrected bibtex reference and paper name.

ngoodger opened the pull request:
Hi

I mistakenly forgot a comma in the bibtex reference causing citations to appear incorrectly and the filename to be incorrect.

ngoodger committed:
Corrected bibtex reference and paper name.

lawrennd reviewed (changes requested):
Thanks Nikolaj, although this is the source material. It doesn’t effect the page. You need to change tje relevant file in the posts subdirectory. Also could you mention why/how PDF is updated?

ngoodger committed:
Updated filename and metadata in _posts.

ngoodger commented:
> Thanks Nikolaj, although this is the source material. It doesn’t effect the page. You need to change tje relevant file in the posts subdirectory. Also could you mention why/how PDF is updated?



> Thanks Nikolaj, although this is the source material. It doesn’t effect the page. You need to change tje relevant file in the posts subdirectory. Also could you mention why/how PDF is updated?

Hi Neil, thanks so much for taking a look at this.
I have now renamed and corrected the metadata in the relevant `_posts` directory as well.  The PDF is completely unchanged, I just renamed it.

lawrennd reviewed (approved):
Brilliant, thank you!
